test/StorageManager.test.js

```diff
@@ -621,4 +621,76 @@ contract('StorageManager', ([Provider, Consumer, randomPerson]) => {
       await expectUtilizedCapacity(0)
     })
   })
+
+  describe('Pausable', () => {
+    it('should not be able to create offer when paused', async () => {
+      await storageManager.pause({ from: randomPerson })
+      expect(await storageManager.paused()).to.be.eql(true)
+      const msg = [padRight(asciiToHex('some string'), 64), padRight(asciiToHex('some other string'), 64)]
+      await expectRevert(
+        storageManager.setOffer(1000, [[10, 100], [20, 100]], [[10, 80], [20, 80]], [constants.ZERO_ADDRESS, token.address], msg, { from: Provider }),
+        'Pausable: paused'
+      )
+    })
+    it('should not be able to to set capacity when paused', async () => {
+      await storageManager.setOffer(1000, [[10, 100], [20, 100]], [[10, 80], [20, 80]], [constants.ZERO_ADDRESS, token.address], [], { from: Provider })
+      await storageManager.pause({ from: randomPerson })
```

Comment:
would rename the varilable as `randomPerson` looks like somebody that should NOT be allowed to pause. Maybe we can create an `owner` variable and use that for the owner in the tests to make it more clear when reading.

Reply:
Done